tests/vcs/git/GitCommitBearTest.py

```diff
@@ -271,11 +271,12 @@ def test_body_checks(self):
 
         # Testing body_regex
         self.git_commit('Shortlog\n\n'
-                        'First line, blablablablablabla.\n'
-                        'Another line, blablablablablabla.\n'
-                        'Fix 1112')
+                        'TICKER---BEGIN\n'
+                        'Bla bla bla\n'
+                        'Fix 1112\n'
+                        'TICKER---END')
         self.assertEqual(self.run_uut(
-                             body_regex=r'Fix\s+[1-9][0-9]*\s*'), [])
+                             body_regex=r'^TICKER---BEGIN.*TICKER---END$'), [])
```

Comment:
Actually, I changed this only because the two test cases already in there would be the same, if  I leave the test as is.